src/cljs/athens/main/core.cljs

```diff
@@ -26,14 +26,49 @@
     (.. ^js @main-window -webContents (send text))))
 
 
+(defn init-electron-handlers
+  []
+  (let [toggle-max-or-min-win-channel "toggle-max-or-min-active-win"
+        close-win-channel "close-win"
+        exit-fullscreen-win-channel "exit-fullscreen-win"]
```

Comment:
could we share the definition of these constants with src/cljs/athens/electron.cljs? i.e., have them defined in one module and import them where needed.